libs/SalesforceSwiftSDK/SalesforceSwiftSDK/Extensions/SalesforceSDKManagerExtensions.swift

```diff
@@ -24,19 +24,20 @@
  */
 import Foundation
 import SalesforceSDKCore
+import SmartSync
 import PromiseKit
 
 extension SalesforceSDKManager {
```

Comment:
Should this class be extensions on `SalesforceSwiftSDKManager` instead, thereby imposing usage of that class as a requirement?

Reply:
not a bad idea :) Let me do that.